clcache.py

```diff
@@ -1265,16 +1268,20 @@ def postprocessHeaderChangedMiss(
 def postprocessNoManifestMiss(
         cache, objectFile, manifestSection, manifestHash, baseDir, sourceFile, compilerResult, stripIncludes):
     returnCode, compilerOutput, compilerStderr = compilerResult
-    listOfIncludes, compilerOutput = parseIncludesList(compilerOutput, sourceFile, baseDir, stripIncludes)
+    listOfIncludes, compilerOutput = parseIncludesList(compilerOutput, sourceFile, stripIncludes)
 
     manifest = None
     cachekey = None
 
     if returnCode == 0 and os.path.exists(objectFile):
         # Store compile output and manifest
-        manifest = Manifest(listOfIncludes, {})
-        includesContentHash = ManifestsManager.getIncludesContentHashForFiles(
-            [expandBasedirPlaceholder(include, baseDir) for include in listOfIncludes])
+        if baseDir:
+            baseDir = os.path.normcase(baseDir)
```

Comment:
Can we norm the basedir at the point where we read it from user input? There we already do some normalization

```python
baseDir = os.environ.get('CLCACHE_BASEDIR')
        if baseDir and not baseDir.endswith(os.path.sep):
            baseDir += os.path.sep
```